packages/filters/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "dev": "run-p build:watch",
     "build": "vite build",
+    "build:ci": "vite build",
```

Comment:
what's the purpose of a separate `build:ci` command that runs the same command as `build`?

Reply:
It's really just a standardised placeholder for _"what's the minimum to build this for production"_. There are things we do in build that aren't needed for a production build. For example, we generate types for packages as part of the usual build, but that's just wasted effort for a production build. Currently only the `ui` lib does anything different, but I think it's a good pattern.